package/tailscale-systemd/package

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 # Copyright (c) 2020 The Toltec Contributors
-# SPDX-License-Identifier: MIT
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This is the license for this file, not the underlying software it's packaging. It must be MIT.